resources/views/snippets/_seo.antlers.html

```diff
@@ -149,40 +149,44 @@
+        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start': new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','{{ seo:google_tag_manager }}')</script>
     {{ elseif seo:tracker_type == 'gtag' }}
         <script async src="https://www.googletagmanager.com/gtag/js?id={{ seo:google_analytics }}"></script>
-        <script>window.dataLayer = window.dataLayer || [];function gtag(){dataLayer.push(arguments);}gtag('js', new Date());gtag('set', new Date());gtag('config', '{{ seo:google_analytics }}' {{ if seo:anonymize_ip }}, {'anonymize_ip': true}{{ /if }});</script>
-    {{ elseif seo:tracker_type == 'scripts' && !seo:use_cookie_banner }}
+        <script>gtag('js', new Date());gtag('set', new Date());gtag('config', '{{ seo:google_analytics }}' {{ if seo:anonymize_ip }}, {'anonymize_ip': true}{{ /if }});</script>
+    {{ elseif seo:tracker_type == 'scripts' && !seo:use_consent_banner }}
         {{ seo:scripts }}
             {{ script_fragments }}
                 {{ if type == 'script_tag' }}
-                    <script {{ async ?= 'async' }} {{ defer ?= 'defer' }} src="{{ source }}" {{ inline_script_attributes }}{{ attribute | slugify }}{{ attribute_value ?= '="{ attribute_value | entities }"' }}{{ !last ?= ' ' }}{{ /inline_script_attributes }}></script>
+                    <script {{ async ?= 'async' }} {{ defer ?= 'defer' }} src="{{ source }}" {{ script_attributes }}{{ attribute | slugify }}{{ attribute_value ?= '="{ attribute_value | entities }"' }}{{ !last ?= ' ' }}{{ /script_attributes }}></script>
                 {{ else }}
-                    <script {{ inline_script_attributes }}{{ attribute | slugify }}{{ attribute_value ?= '="{ attribute_value | entities }"' }}{{ !last ?= ' ' }}{{ /inline_script_attributes }}>{{ inline_script }}</script>
+                    <script {{ script_attributes }}{{ attribute | slugify }}{{ attribute_value ?= '="{ attribute_value | entities }"' }}{{ !last ?= ' ' }}{{ /script_attributes }}>{{ inline_script }}</script>
                 {{ /if }}
             {{ /script_fragments }}
         {{ /seo:scripts }}
     {{ /if }}
 
-    {{ if seo:use_cookie_banner && (seo:tracker_type == 'gtm' || seo:tracker_type == 'gtag') }}
-        <script>
-            gtag('consent', 'default', {
-                'ad_storage': 'denied',
-                'analytics_storage': 'denied',
-                'wait_for_update': 500
-            });
-            dataLayer.push({
-                'event': 'default_consent'
```

Comment:
Is that event not needed anymore? It's missing above

Reply:
I didn't see it in the google docs anymore.

Reply:
https://developers.google.com/tag-platform/security/guides/consent

So I think we're all good!